torch/csrc/cuda/shared/cudart.cpp

```diff
@@ -31,6 +31,12 @@ void initCudartBindings(PyObject* module) {
   // HIP rewrite rules from changing these names when building with HIP.
 
 #if !defined(USE_ROCM)
+#if CUDA_VERSION >= 12000
+  enum __device_builtin__ cudaOutputMode_t {
+    cudaKeyValuePair = 0x00, // Output mode Key-Value pair format.
+    cudaCSV = 0x01 // Output mode Comma separated values format.
+  };
+#endif
```

Comment:
`cudaOutputMode_t` is used in `cudaProfilerInitialize` only. The latter is gone in CUDA 12, so there is probably no need to define this enum.